.github/workflows/hlint.yml

```diff
@@ -1,20 +1,21 @@
 name: server lint
 on:
+  label:
   pull_request:
     paths:
       - 'server/**'
 
 jobs:
   hlint:
     runs-on: ubuntu-20.04
+    if: !contains(github.event.pull_request.labels.*.name, 'ignore-lint-checks')
```

Comment:
calling it `hlint` will disambiguate from other lints?

Reply:
SG! Sorry, this is still a draft, I'm testing that this works as expected, the documentation for actions is a bit lackluster. ^^

Reply:
I might settle for `ignore-server-lint-checks`, since I'm working on other lint steps that are not hlint I'd like to be able to disable if necessary.